README.md

```diff
@@ -0,0 +1,56 @@
+# Financial App Admin
+
+## Setup
+
+Use NodeJS 16.20.2. (We recommend using `nvm` to manage multiple NodeJS versions.)
+
+To install packages:
+
+```
+npm install
+```
```

Comment:
It would be nice if we could add more information on what this repo is actually used for

Reply:
Oh definitely. I'll add more after I figure out the rest of the details. (Or someone else can if they like.)